app/code/Magento/CatalogGraphQl/Model/Product/Option/DateType.php

```diff
@@ -77,4 +79,18 @@ public function useCalendar()
     {
         return false;
     }
+
+    /**
+     * Return option value for quote option
+     *
+     * @param string $optionValue Prepared for cart option value
+     * @return string
+     */
+    public function getFormattedOptionValue($optionValue)
```

Comment:
is the value coming from a config?
I wonder what's wrong with the original method?
I also remember that we don't actually format dates in graphql...

Reply:
HI Chris, yeah... It was something we discussed before, and there were objections: some times we need locale and formatting that are done on backend, so there are only 2 options: send all info via GraphQL to front or do it on backend with just additional data... I think do it on backend is better

Reply:
No, the issue is that the original method returns a date in another format:
input value - "01-01-01 02:00:05";
output value - "1/2/01, 2:00 AM".
Should be the same.

Reply:
Well this will be a BiC (backward incompatible change), as discussed to @paliarush 
there's no place in Graphql where we actually chose to format dates, as this is exposed in the storeConfig
So we expose dates in GMT and mysql format

I think we should close this ticket @nrkapoor